### DISCUSSION TITLE
Is "C-style comments" accurate, given no support for multiline?

### DISCUSSION THREAD

**Discussion by cueckoo** [cue-lang/cue#858] [General]
_Originally opened by @NReilingh in https://github.com/cuelang/cue/discussions/858_

When I read the CUE tutorial, I took note of the "C-style comments" item listed at https://cuelang.org/docs/tutorials/tour/intro/json/, and interpreted that to mean support for both single-line (`// comment`) and multiline (`/* comment */`) comments.

I was rather surprised, then, when CUE blew up after I tried to feed it some multiline comments. It seems as though only the `//` style is supported.

Is it accurate to call these C-style comments in the tutorial, especially given that C's original comment style was the multiline variation, and the single-line `//` style was actually an extension from C++?

**Comment by cueckoo** [comment]
_Original reply by @myitcv in https://github.com/cuelang/cue/discussions/858#discussioncomment-542080_

I suspect there might have been a time when CUE _did_ support multi-line comments; @mpvl will be able to confirm.

Either way, they are not support today. So @NReilingh I'd agree - we should make the tutorial line up with [the spec](https://tip.cuelang.org/docs/references/spec/#comments).

**Reply by cueckoo** [reply]
_Original reply by @NReilingh in https://github.com/cuelang/cue/discussions/858#discussioncomment-542080_

Cool, I've submitted a PR: https://github.com/cuelang/cue/pull/864

**Comment by deepbrook** [comment]
Hey, @myitcv / @mpvl - I think this was lost during/after the website relaunch? The page at https://cuelang.org/docs/tour/basics/json-superset/ still (again?) states `C-style comments are allowed`. 

Unless you guys are bringing multi-line comments back? ;D that'd be neat.

**Reply by mvdan** [reply]
@deepbrook oops, thanks for spotting. Sent a patch here: https://cue.gerrithub.io/c/cue-lang/cuelang.org/+/1229638
